src/components/ItemCard.js

```diff
@@ -1,10 +1,10 @@
 import "./ItemCard.css";
 
-export default function ItemCard() {
+export default function ItemCard({ image }) {
   return (
     <div className="Item-card">
```

Comment:
Also, as an easy to follow CSS naming convention, you could give the root element of a component always a class name equal to the component name, (in this case, `ItemCard`) and use that as the block in BEM.